### ISSUE TITLE
Grid: Select element in filter row missing text label (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#938]
### Expected behavior
Form input fields have text labels that exist are programmatically associated with the input field.

### Actual behavior
There is no text label. 

### Steps to reproduce

1.  Open filter row in first example
2. Locate dropdown (the one with All values, Option 1...)
3. Find text label that make sense to label the select element
